app/src/main/java/com/hamdan/forzenbook/data/network/LoginService.kt

```diff
@@ -1,16 +1,27 @@
 package com.hamdan.forzenbook.data.network
 
 import retrofit2.Response
+import retrofit2.http.Field
+import retrofit2.http.FormUrlEncoded
 import retrofit2.http.GET
+import retrofit2.http.POST
 import retrofit2.http.Query
 
-interface LoginService {
+private const val EMAIL = "email"
```

Comment:
Companion

Reply:
I was thinking interfaces couldn't have companion objects so I put it outside, implemented a companion object.